client/src/components/EnquiryModal/EnquiryModal.component.tsx

```diff
@@ -39,12 +41,26 @@ export const EnquiryModal = ({
   const { errors: formErrors } = formState
   const [isLoading, setIsLoading] = useState<boolean>(false)
   const [captchaResponse, setCaptchaResponse] = useState<string | null>(null)
+  const googleAnalytics = useGoogleAnalytics()
+  const agencyEmail = agency.email
+
+  const sendSubmitEnquiryEventToAnalytics = (agencyEmail: string) => {
+    googleAnalytics.sendUserEvent(
+      googleAnalytics.GA_USER_EVENTS.SUBMIT_ENQUIRY,
+      agencyEmail,
+    )
+    FullStory.event(googleAnalytics.GA_USER_EVENTS.SUBMIT_ENQUIRY, {
+      // property name uses `ident_type` pattern
```

Comment:
```suggestion
```

Reply:
Comment addressed in commit https://github.com/opengovsg/askgovsg/pull/249/commits/0abdfd99aa6fb009a6b871b4059df30c6d876026.